tests/pipelines/test_pipelines_text2text_generation.py

```diff
@@ -124,3 +124,31 @@ def test_small_model_tf(self):
         # do_sample=False necessary for reproducibility
         outputs = generator("Something there", do_sample=False)
         self.assertEqual(outputs, [{"generated_text": ""}])
+
+    @require_torch
+    def test_return_dict_in_generate(self):
+        greedy_search_inference_config = {"do_sample": False, "num_return_sequences": 1, "num_beams": 1}
+        pipe = pipeline(task="text2text-generation", model="patrickvonplaten/t5-tiny-random")
+        pipe("hello", return_dict_in_generate=True, **greedy_search_inference_config)
```

Comment:
Do you mind adding a check for those returned values?

Reply:
done!